content/03.methods.md

```diff
@@ -48,6 +48,14 @@ We trained this model using scikit-learn's [@raw:scikit-learn] implementation of
 For each principal component we calculated its cosine similarity with  all tokens in our word2vec model's vocabulary.
 We report the top 100 positive and negative scoring tokens in the form of  word clouds, where the size of each word corresponds to the magnitude of similarity and color represents positive (blue) or negative (orange) association.
 
-### Recommending Journals/ bioRxiv Audience Analysis
-1. This title will update as analysis is completed
-2. This section will describe how the above process is conducted
+### Journal Recommendation
+We trained two k-nearest neighbor classifiers to recommend journals based on article content.
+The first classifier recommends journals based on articles in close proximity to the query and the second classifier recommends journals based on close proximity to each journal's centroid.
```

Comment:
```suggestion
The first classifier recommended journals based on the articles in close proximity to the query article and the second classifier recommended journals based on the journal centroids in close proximity to the query article.
```
What do you think of this phrasing? I switched it to past tense & tried to make "query article" consistent. I think it's not quite right and needs further adjustment though.